src/utility/QueryStore.js

```diff
@@ -18,6 +18,19 @@ export default class QueryStore {
     );
   }
 
+  edit(item) {
+    const index = this.items.findIndex(
```

Comment:
nit: we could just merge this with the below `if` statement, or come up with a more distinguished variable name than `index`